### PR TITLE
fix title

### PR DESCRIPTION
**Pull request by menthy-wu**

### CODE REVIEW

File: src/components/dynamic/admin/Title.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
-const Title = ({ title, classes = "text-2xl font-extrabold pr-[3%]" }) => {
```
Comment (shahdivyank) [inline src/components/dynamic/admin/Title.jsx:1]: did u remove the prop classes from the component calls

Comment (menthy-wu) [reply src/components/dynamic/admin/Title.jsx:1]: yea

Comment (menthy-wu) [reply src/components/dynamic/admin/Title.jsx:1]: oh lol i didn't push it

Comment (menthy-wu) [reply src/components/dynamic/admin/Title.jsx:1]: oh nvm i did push it

**Review by shahdivyank — APPROVED** [review]

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**